oak_functions/examples/key_value_lookup/config.toml

```diff
@@ -0,0 +1,8 @@
+protocol = "Grpc"
+lookup_data_url = "https://storage.googleapis.com/oak_lookup_data/lookup_data_weather"
```

Comment:
Is the use of weather lookup data intentional?

Reply:
Sort of, this example is not used for anything yet, so any data source will do. I will probably change the deployment scripts in the future and generate some more predictable data to use in a demo.